src/smokeTest/java/uk/gov/hmcts/reform/iacaseapi/SmokeTest.java

```diff
@@ -1,12 +1,17 @@
 package uk.gov.hmcts.reform.iacaseapi;
 
+import static io.restassured.RestAssured.*;
 import static org.assertj.core.api.Assertions.assertThat;
 
-import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
 import net.serenitybdd.rest.SerenityRest;
 import org.apache.commons.lang3.StringUtils;
 import org.junit.Test;
 import org.springframework.http.HttpStatus;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestClientException;
 
 public class SmokeTest {
```

Comment:
Let's put a brief comment here.  E.g. The SmokeTest calls only the /health endpoint as this ensures that all downstream dependencies are healthy including the database and downstream api's.

Reply:
Added now